index.js

```diff
@@ -16,12 +16,52 @@ mongoose
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+  .then(() => { // Iteration 2 and 3 
+    console.log('Database has been dropped')
+    Recipe.create({ title: "Pizza", cuisine: "Italian" })
+      //createRecipe
+      .then((response) => {
+        console.log(response)
+      })
+    let recipeMany = Recipe.insertMany(data)
+    recipeMany
+      .then(() => {
+        console.log('all added')
+      })
+    Promise.all([recipeMany]) //iteration 4
```

Comment:
promise.all is mostly used when you want several promises to resolve. In this case, since you only need one to resolve you can put the code inside this .then inside the promise .then on line 30 instead.